events/wc_court_events.txt

```diff
@@ -160,7 +160,12 @@ character_event = {
 		picture = GFX_evt_nraqi
```

Comment:
`WCCOU.200` is still broken.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/108753042-ad031200-755d-11eb-9afc-722fc125d562.png)

</details>